iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -185,7 +185,7 @@ struct DefaultChunkQueueConfig
 
 // alias for cxx::string
 using ConfigFilePathString_t = cxx::string<1024>;
-using ProcessName_t = cxx::string<100>;
+using ProcessName_t = cxx::string<MAX_PROCESS_NAME_LENGTH>;
 using RunnableName_t = cxx::string<100>;
```

Comment:
Why not MAX_PROCESS_NAME_LENGTH as well?

Reply:
I'd rather create a static `capacity` function in `cxx::string` than doing this. Is this okay for you?

Reply:
No please don't. Again here the argument is that all settings are stored in the posh_types. Maybe we support at some point in the future a different process name length. Every setting is then stored in posh_types except this one which lives in some header file.

But a static capacity function is a good idea anyway!

Reply:
@elfenpiff the alias for `ProcessName_t` also lives in the posh_types. It comes down to go to posh_types and either change the constexpr or the template parameter of the alias.